### ISSUE TITLE
How to use these codes?

### ISSUE THREAD

**Issue by HandsomeCBX** [FeipengMa6/PriSA#1]
Hello, can you tell me more details about processing these codes? Thank you very much.

**Comment by FeipengMa6** [comment]
Hi, I have updated the readme. Now you can run the code by following the provided instructions.

**Comment by HandsomeCBX** [comment]
> Hi, I have updated the readme. Now you can run the code by following the provided instructions.

Thank you very much,I am downloading datasets.I will try to do it.Thank you!!

**Comment by HandsomeCBX** [comment]
> Hi, I have updated the readme. Now you can run the code by following the provided instructions.

Hi,Please tell me need I create pretrain directory by myself? I do not konw how to download bert model.Thanks for your reply.

**Comment by HandsomeCBX** [comment]
> Hi, I have updated the readme. Now you can run the code by following the provided instructions.

Need I download all of codes?
![image](https://github.com/FeipengMa6/PriSA/assets/148057185/7cb295fd-2440-46d0-b532-0b3c5d9e41fe)

**Comment by FeipengMa6** [comment]
Yes, you can download files of this repo manually, or you can use the following command,
```bash
cd ./pretrained_models
git lfs install
git clone https://huggingface.co/bert-base-uncased
```
I hope this will help you!

**Comment by HandsomeCBX** [comment]
> Yes, you can download files of this repo manually, or you can use the following command,
> 
> ```shell
> cd ./pretrained_models
> git lfs install
> git clone https://huggingface.co/bert-base-uncased
> ```
> 
> I hope this will help you!



> Yes, you can download files of this repo manually, or you can use the following command,
> 
> ```shell
> cd ./pretrained_models
> git lfs install
> git clone https://huggingface.co/bert-base-uncased
> ```
> 
> I hope this will help you!
Thank you, but there is no pretrained_models,need I create it manually?

**Comment by HandsomeCBX** [comment]
> Yes, you can download files of this repo manually, or you can use the following command,
> 
> ```shell
> cd ./pretrained_models
> git lfs install
> git clone https://huggingface.co/bert-base-uncased
> ```
> 
> I hope this will help you!

Hi,there is a problem,after python runx.runx
![811a57073a56406f4979ccaec1ef77a](https://github.com/FeipengMa6/PriSA/assets/148057185/612aebd1-b8f7-4e8f-9a8c-ae5b9e3cb283)
